docs/hosted-orchestration/running-tests.md

```diff
@@ -0,0 +1,202 @@
+        entrypoint: "mvn test"
+        files:
+          - src: "runsauce.json"
+            dst: "/workdir/runsauce.json"
+        artifacts:
+          - "/path/inside/container/file.log"
+        env: 
+          KEY: value
+  ```
+
+  Then run with
+
+  ```bash
+    saucectl run
+  ```
+
+  You should receive a successful output that looks like
+
+  ```bash
+  ⚡  saucectl run
+     Running version 0.0.0+unknown
+     12:46:56 WRN A new version of saucectl is available (v0.120.1)
+     12:46:56 INF Launching workers. concurrency=1
+     12:46:56 INF Starting suite. image=mikedonovan1987/java-hosted-execution:pr-1 suite="run sauce test"
+     12:46:57 INF Started suite. image=mikedonovan1987/java-hosted-execution:pr-1 runID=056e38a0684a4a99b4478aa73f214e3f suite="run sauce test"
+     12:47:06 INF Suites in progress: 1
+     12:47:16 INF Suites in progress: 1
+     12:47:26 INF Suites in progress: 1
+     12:47:36 INF Suites in progress: 1
+     12:47:46 INF Suites in progress: 1
+     12:47:56 INF Suites in progress: 1
+     12:48:06 INF Suites in progress: 1
+     12:48:16 INF Suites in progress: 1
+     12:48:26 INF Suites in progress: 1
+     12:48:27 INF Suite finished. passed=true runID=056e38a0684a4a99b4478aa73f214e3f suite="run sauce test"
+
+       Name                              Duration    Status       Attempts
+────────────────────────────────────────────────────────────────────────────
+  ✔    run sauce test                       1m30s    Succeeded           1
+────────────────────────────────────────────────────────────────────────────
+  ✔    All suites have passed               1m30s
+  ```
+  </TabItem>
+</Tabs>
+
+## Configuration Details
+
+Each of the properties supported for running Hosted Orcheestration tests through `saucectl` is defined below.
+
+## `apiVersion`
+
+<p><small>| REQUIRED | STRING |</small></p>
+
+Identifies the version of the underlying configuration schema. At this time, `v1alpha` is the only supported value.
+
+```yaml
+apiVersion: v1alpha
+```
+
+---
+
+## `kind`
+
+<p><small>| REQUIRED | STRING |</small></p>
+
+Tells SauceCTL this is a Hosted Orcheastration job. `imagerunner` is the required value.
+
+```yaml
+kind: imagerunner
+```
+
+---
+
+## `image`
+
+<p><small>| REQUIRED | STRING |</small></p>
+
+The location of your Docker image. Takes the format [registry]/[image]:[tag]
+
+```yaml
+image: saucelabs/sl-demo-docker-primary:0.0.1
+```
+
+---
+
+## `imagePullAuth`
+
+<p><small>| OPTIONAL | OBJECT |</small></p>
+
+The credentials needed to access an image hosted in a private register
+
+```yaml
+imagePullAuth: 
+  user: sample_user
+  token: sample_token
+```
+
+---
+
+## `entrypoint`
+
+<p><small>| REQUIRED | STRING |</small></p>
+
+The command the execute once the container is ready.
+
+```yaml
+entrypoint: mvn test
+```
+
+---
+
+## `files`
+
+<p><small>| OPTIONAL | ARRAY |</small></p>
+
+Files to be uploaded onto the container. Useful for populating test data that your scripts access.
```

Comment:
A few things I'd want to up clarified:
1. How do you specify the list files? Is it a comma separated list of strings?
2. Does it have to be just files or can it include directories? Can it include globs?
3. For the `dst` do you have to specify an absolute path (like `/workdir/runsauce.json`) or can you give it a relative path? And if it is a relative path, will that be the same as the `WORKDIR`?